source/gems/dry-types/built-in-types.html.md

```diff
@@ -29,6 +29,20 @@ Base definitions:
 - `Types::Array`
 - `Types::Hash`
 
+All types are aliased under the `Strict` namespace:
```

Comment:
D'oh, on closer inspection of the source code I realise that these aren't actually "aliases", and e.g. `Types::Strict::Int` and `Types::Int` aren't functionally the same. I'll fix the PR tomorrow; disregard for now.